vars/piperPipelineStageBuild.groovy

```diff
@@ -61,5 +67,17 @@ void call(Map parameters = [:]) {
             testsPublishResults script: script, junit: [updateResults: true]
             checksPublishResults script: script
         }
+
+        if (config.mavenExecuteStaticCodeChecks) {
```

Comment:
I assume both mavenExecuteStaticCodeChecks and npmExecuteLint won't be enabled in the GPP by default, right? Could GPP users benefit from it if they want?

Reply:
Exactly they are not enabled by default. We enable them for the Cloud SDK Pipeline using the conditions in `cloudSdkStageDefaults.yml`
This could also be added to the GPP, if the same (or different) conditions are added to the `stageDefaults.yml`, or if the users overwrite the runStage/runStep maps.